### DISCUSSION TITLE
Reduce the spine title font size

### DISCUSSION THREAD

**Discussion by js-vieira** [joaomlourenco/novathesis#423] [Q&A]
Hello,

I hope you're doing well.

My dissertation's title does not fit properly in the spine (it's a little bit on top of the author's box). Is it possible to reduce the font size of the title in the template's spine? If so, what is the best way to do that?

Thank you in advance.

**Comment by joaomlourenco** [comment]
The font size is calculated automatically so that the title fits properly in the available space.  Of course it can be customzied, but should no be necessary.
Which version of the template are you using? Which university/school/degree?

**Reply by js-vieira** [reply]
I'm using the version 6.10.5 for FCT/NOVA/master’s degree.
Through one of your answers that I saw in the Discussions, I was able to discover the file `NOVAthesisFiles/Schools/nova/fct/nova-fct-defaults.ldf`, where I tried to replace 
`\spine(box,date,len)={1.5cm} `
by 
`\spine(box,date,len)={1.1cm}` 
and it worked. 
I’m not sure if it is the best solution, but I didn't want to compromise your code or mess it up.

**Reply by joaomlourenco** [reply]
Thanks for the suggestion.  I'll look it up.
